### DISCUSSION TITLE
Clipboard cannot be used when login to Windows

### DISCUSSION THREAD

**Discussion by bero** [gravitational/teleport#40072] [Q&A]
I  am a member in a team that have Windows in GCP (Google Cloud Platform) and I use Teleport.
I use Ubunt0 23.10 as OS and KVM 8.0.4 as virtualization.
GCP is Windows Server 2022 Datacenter version 21H2.

Generally it works well but there are some problems. To get discussion more focused I only write one problem in each thread.

When login with Teleport I use Edge in a Windows 10 guest VM. Clipboard not possible to use in Windows login for password in GCP. I have to type password manually. Is this by design or a bug ?

**Comment by webvictim** [comment]
Teleport doesn't prevent any use of the Windows clipboard in specific applications - this sounds like it might be a Google restriction instead.

Some questions:
- What browser are you using on the Ubuntu (client) side?
- Have you tried a different browser on the Windows side?
- Does the clipboard work correctly in other Windows applications?
- Does the clipboard work with other websites that are not GCP?

Thanks!
